applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -465,8 +465,7 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
     const ProcessInfo& rCurrentProcessInfo
     )
 {
-    const GeometryType::IntegrationMethod integration_method =
-        GetGeometry().GetDefaultIntegrationMethod();
+    const GeometryType::IntegrationMethod integration_method = GetGeometry().GetDefaultIntegrationMethod();
```

Comment:
changes in the base_solid_element should go in and be discusses as a different PR